### PR TITLE
endpoint: do not export additional fields

### PR DESCRIPTION
**Pull request by ianvernon**
The following fields are now hidden:

* Container ID
* Docker Endpoint ID
* Docker Network ID
* Container Name

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/9170)

### CODE REVIEW

**Comment by ianvernon** [comment]
test-me-please

File: pkg/workloads/docker.go [hunk 1]
```diff
@@ -221,7 +221,7 @@ func (d *dockerClient) IsRunning(ep *endpoint.Endpoint) bool {
 			runtimeRunning = true
 			found := false
 			for _, v := range nls.Containers {
-				if v.EndpointID == ep.DockerEndpointID {
+				if v.EndpointID == ep.GetDockerEndpointID() {
```
Comment (aanm) [inline pkg/workloads/docker.go]: to avoid doing multiple rlocks, get the dockerendpointID before the for loop

**Review by aanm — COMMENTED** [review]
Small nit

**Review by joestringer — APPROVED** [review]
Looks good to me.

File: pkg/logging/logfields/logfields.go [hunk 2]
```diff
@@ -38,7 +38,7 @@ const (
 	// EventUUID is an event unique identifier
 	EventUUID = "eventID"
 
-	// ContainerID is the container identifier
+	// containerID is the container identifier
```
[the remark this answers is not in the document]
Comment (vadorovsky) [reply pkg/logging/logfields/logfields.go]: That comment seems to be relevant - probably your `sed` command touched the file which should not be touched ;)

**Review by vadorovsky — CHANGES_REQUESTED** [review]
One nit, but overall looks good

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
Istio pods never got ready: 	

```
istio-system   istio-cleanup-secrets-1.2.5-66bwp         0/1       Completed   0          4m        10.10.1.185     k8s2      <none>
	 istio-system   istio-galley-7fcdcc57f9-whs2j             1/1       Running     0          4m        10.10.1.154     k8s2      <none>
	 istio-system   istio-ingressgateway-597fd5bd96-pg445     0/1       Running     0          4m        10.10.1.15      k8s2      <none>
	 istio-system   istio-init-crd-10-726hh                   0/1       Completed   0          4m        10.10.1.103     k8s2      <none>
	 istio-system   istio-init-crd-11-x95zg                   0/1       Completed   0          4m        10.10.1.21      k8s2      <none>
	 istio-system   istio-init-crd-12-jctdf                   0/1       Completed   0          4m        10.10.1.96      k8s2      <none>
```

**Comment by ianvernon** [comment]
test-me-please

**Review by aanm — APPROVED** [review]

**Pull request merged**